content/contribute/index.md

```diff
@@ -0,0 +1,133 @@
+---
+
+So you want to contribute to IPFS and the ecosystem? Here is a quick listing
+of everything we need help with and how you can get started helping out. Please
+note that even though what you want to do is not listed here, we probably accept
+contributions for it! If you're unsure, please open a issue.
+
+## Areas of contribution
+
+- [Code](#code)
+- [Documentation](#documentation)
+- [Support](#support)
+- [Testing](#testing)
+- [Design](#design)
+- [Issues / Triaging](#issues-triaging)
+- [Community](#community)
+- [Applications](#applications)
+- [Protocol Design](#protocol-design)
+- [Research](#research)
+
+### Code
+
+IPFS and it's sister-projects are big projects with lots of code, in multiple
+languages. We always need help with writing and maintaining code, but it can
+be daunting to just jump into it. Therefore, we have made sure that we label
+features or bugfixes that people can help out with, with "Help Wanted" label.
+That's a excellent place for you to start contributing code.
+
+The biggest and most active repositories we have today are:
+
+- https://github.com/ipfs/go-ipfs
+- https://github.com/ipfs/js-ipfs
+- https://github.com/libp2p/go-libp2p
+- https://github.com/libp2p/js-libp2p
+
+If you want to start contributing to the core of IPFS, those repositories are
+a great start. But the "Help Wanted" label exists in all of our repositories
+(in the Github organizations IPFS / libp2p / IPLD / Multiformats)
+
+### Documentation
+
+Again, IPFS is a huge project and undertaking. With lots of code comes the need
+of having lots of good documentation too! However, this is currently are area
+we're severly lacking enough manpower to make great. If writing technical documentation is your area, we'd love your help!
+
+Best place to get started is by looking through the Github Issues over at: https://github.com/ipfs/docs
+
+### Support
+
+IPFS already have lots of users and other curious people experimenting and using
+IPFS for their applications. These users sometimes get stuck or have questions
+that needs answering. If you've contributed something with code or documentation,
+chances are that you probably can help a lot of people with their questions.
+
+We mostly do support via the forum we have here: https://discuss.ipfs.io/
+
+### Testing
+
+As we're continously improving IPFS everyday, mistakes can happen and we release
+something that could work much better or simply doesn't work. If you like to
+dig into edge-cases or writing testing scenarios, getting dirty with all our
+testing infrastructure would be a wet dream for you.
+
+Mainly, CI work is happening over at https://github.com/ipfs/jenkins and larger
+scale tests are planned to be happening over at https://github.com/ipfs/kubernetes-ipfs
+
+
+### Design
+
+Currently, there is no fulltime designers working with IPFS and it's sister-projects.
+Although it would be nice to have, we still have many smaller pieces that can be
+contributed by you as a outsider!
```

Comment:
Should we be linking to any of these projects here?

- https://github.com/ipfs-shipyard/ipfs-webui
- https://github.com/ipfs-shipyard/ipfs-react-components
- https://github.com/ipfs-shipyard/ipfs-ui-style-guide
- https://github.com/ipfs-shipyard/ipfs-css
- https://github.com/ipfs-shipyard/pm-ipfs-gui

Reply:
Yeah, makes a lot of sense. @alanshaw, @lidel and @akrych might know more projects that needs help from designers as well. I think ipfs-desktop should be here as well.

Reply:
Safe set of design-related repos:
  - https://github.com/ipfs-shipyard/pm-ipfs-gui
  - https://github.com/ipfs-shipyard/ipfs-ui-style-guide (it will link / reference other repos)

Apps that are under "gui" umbrella: 
  - https://github.com/ipfs-shipyard/ipfs-desktop
  - https://github.com/ipfs-shipyard/ipfs-companion
  - https://github.com/ipfs-shipyard/ipfs-webui